api/models/v0.2/modelResponse.js

```diff
@@ -10,26 +10,18 @@ const url = require('url');
  */
 function modelPatronCreatorResponse(responseData, status, requestData) {
   return {
-    data: [
-      {
-        id: parseInt(responseData.link.split('/').pop(), 10),
-        names: requestData.names,
-        barcodes: requestData.barcodes,
-        expirationDate: '',
-        birthDate: requestData.birthDate,
-        emails: requestData.emails,
-        pin: requestData.pin,
-        patronType: parseInt(requestData.patronType, 10),
-        patronCodes: requestData.patronCodes,
-        blockInfo: requestData.blockInfo,
-        addresses: requestData.addresses,
-        phones: requestData.phones,
-      },
-    ],
-    count: 1,
-    totalCount: 0,
-    statusCode: status,
-    debugInfo: [],
+    id: parseInt(responseData.link.split('/').pop(), 10),
```

Comment:
If parseInt fails here or on parsing requestData.patronType, then modelPatronCreatorResponse will crash.  Is this the best course of action, or would a try/catch block be called for?